mephisto/server/architects/router/deploy/server.js

```diff
@@ -476,6 +479,20 @@ app.post('/submit_task', upload.any(), function(req, res) {
   }
 });
 
+app.post('/log_error', function(req, res) {
+  var provider_data = req.body;
+  let agent_id = provider_data.USED_AGENT_ID;
+  delete provider_data.USED_AGENT_ID;
```

Comment:
tip: one way you can condense the functionality of the above lines is to do the following:

`const { USED_AGENT_ID: agent_id, ...provider_data} = req.body`

this will end up doing the same thing as the above but in a single line. what it does is destructures `req.body` storing `USED_AGENT_ID` in `agent_id` and then capturing the remaining properties in `provider_data` using the spread operator.